tests/data/GlyphsUnitTestSans.glyphs

```diff
@@ -115,6 +115,9 @@ GSOffsetVertical = 9;
 GSRoughenHorizontal = 15;
 GSRoughenSegmentLength = 15;
 GSRoughenVertical = 10;
+com.schriftgestaltung.Glyphs.ufoData = {
```

Comment:
can this occur in real life?

Reply:
We've seen binary data in our Glyphs files that was a remain of old Robofont plugin data... I don't think the data in there was worth anything or taken into account by our build tools, but it did appear and caused a crash of glyphsLib for one or two projects, that's why we want glyphsLib to handle it. (was that your question?)

Reply:
kind of, yeah. Ideally I would like to know all the places where this binary data can (or is likely to) happen in a glyphs file. I guess only @schriftgestalt can tell for certain.

Reply:
Binary data can occur in userData and in custom parameters (if some script or plugin puts it there). In custom parameters it is not officially supported but it uses the same code path as the storage of user data.